ta/arch/arm/user_ta_header.c

```diff
@@ -25,26 +25,44 @@ const char trace_ext_prefix[]  = "TA";
 /* exprted to user_ta_header.c, built within TA */
 struct utee_params;
 
+#if defined(ARM32) && defined(CFG_UNWIND)
```

Comment:
This is doesn't look very nice, how about moving it into an `_a32.S` file?

Reply:
Agreed. I will clean this up.